src/functionalTest/resources/application.yaml

```diff
@@ -0,0 +1,53 @@
+management:
+  endpoint:
+    health:
+      show-details: "always"
+  endpoints:
+    web:
+      base-path: "/"
+      exposure:
+        include: health, info, prometheus
+  security:
+    enabled: false
+
+spring:
+  application:
+    name: Bulk Scan Payment Processor
+  main:
+    # without this, Spring can't properly resolve feign clients when IDAM and CCD clients are both included
+    # TODO: remove when possible
+    allow-bean-definition-overriding: true
+
+azure:
+  servicebus:
+    payments:
+      connection-string: ${PAYMENTS_QUEUE_READ_CONNECTION_STRING}
+      write-connection-string: ${PAYMENTS_QUEUE_WRITE_CONNECTION_STRING}
+      queue-name: payments
+      max-delivery-count: 5
+
+idam:
+  s2s-auth:
+    url: ${S2S_URL:http://localhost:4552}
+    secret: ${S2S_SECRET}
+    name: ${S2S_NAME:bulk_scan_payments_processor}
+  api:
+    url: ${IDAM_API_URL}
+  client:
+    id: 'bsp'
```

Comment:
Do you need all these properties in functional tests?

Reply:
Yes, failing otherwise. Func test needs access to IDAM